java/server/src/com/appdynamics/wpt/WptHookAwareHandler.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) AppDynamics Inc
+ * All rights reserved
+ */
+package com.appdynamics.wpt;
+
+import org.openqa.selenium.remote.DriverCommand;
+import org.openqa.selenium.remote.Command;
+
+import java.io.IOException;
+import java.util.logging.Logger;
+import java.net.MalformedURLException;
+
+/**
+ * @author <a mailto="renault.lecoultre@appdynamics.com">Renault John Lecoultre</a>
+ * @since 6/26/2015
+ *
+ * @author <a mailto="karthik3186@gmail.com">Karthik Krishnamurthy</a>
+ * @since 2/14/2015
+ */
+public class WptHookAwareHandler {
+  private WptHookClient wptHookClient;
```

Comment:
Not sure it's possible. The `WptHookClient` constructor requires parameters provided to the `WptHookAwareHandler` constructor. I'm not aware of a way to have a final object initialized in a constructor. Is that possible in Java?